arch/arm/soc/st_stm32/stm32f4/soc_pinmux.c

```diff
@@ -373,6 +434,8 @@ static const struct stm32_pinmux_conf pins[] = {
 	STM32_PIN_CONF(STM32_PIN_PE8, pin_pe8_funcs),
 
 	/* Port F */
+	STM32_PIN_CONF(STM32_PIN_PF0, pin_pf0_funcs),
```

Comment:
Port F/G is not supported on F401RE.
I think a #ifdef GPIOX should be added for these two ports.
I might not break anything, but at least use space uselessly